Library/Homebrew/resource.rb

```diff
@@ -277,6 +280,28 @@ def detect_version(val)
     version unless version.null?
   end
 
+  def determine_url_mirrors
+    extra_urls = []
+
+    # glibc-bootstrap
+    if url.start_with?("https://github.com/Homebrew/glibc-bootstrap/releases/download")
+      tag, filename = url.split("/")[-2..]
```

Comment:
```suggestion
      tag, filename, = *url.split("/")
```

or whatever the correct version is to avoid `[-2..]`